tf2onnx/onnx_opset/math.py

```diff
@@ -54,62 +54,76 @@ def version_9(cls, ctx, node, **kwargs):
         pass
 
 
+def make_minmax_op(ctx, op_type, inputs, outputs,
+                   output_shapes=None, output_dtypes=None):
+    # support more dtype
+    supported_dtypes = [
```

Comment:
why only support these dtype, could we add some comment in case when needing to add more dtype supported?

Reply:
it's supported dytypes of ONNX op not convertor supported dtype